File: android/api-tests/src/test/java/com/revenuecat/apitests/java/OfferingsMapperApiTests.java
package com.revenuecat.apitests.java.mappers;

import com.revenuecat.purchases.Offerings;
import com.revenuecat.purchases.hybridcommon.mappers.OfferingsMapperKt;

import java.util.Map;

import kotlin.Unit;
import kotlin.jvm.functions.Function1;

@SuppressWarnings("unused")
class OfferingsMapperApiTests {
    private void checkMapAsync(Offerings offerings, Function1<? super Map<String, ?>, Unit> callback) {
        OfferingsMapperKt.mapAsync(offerings, callback);
    }
}

File: android/api-tests/src/test/java/com/revenuecat/apitests/kotlin/mappers/OfferingsMapperApiTests.kt
package com.revenuecat.apitests.kotlin.mappers

import com.revenuecat.purchases.Offerings
import com.revenuecat.purchases.hybridcommon.mappers.mapAsync

@Suppress("unused", "UNUSED_VARIABLE")
private class OfferingsMapperApiTests {
    fun checkMapAsync(offerings: Offerings, callback: (Map<String, Any?>) -> Unit) {
        val unit: Unit = offerings.mapAsync(callback)
    }
}
